build/stf-run-ci/defaults/main.yml

```diff
@@ -20,21 +20,21 @@ __internal_registry_path: image-registry.openshift-image-registry.svc:5000
 __deploy_minio_enabled: false
 __loki_skip_tls_verify: false
 __golang_image_path: quay.io/infrawatch/golang:1.16
-__loki_image_path: quay.io/infrawatch/loki:2.2.1
+__loki_image_path: quay.io/infrawatch/loki:2.4.2
 
 sgo_image_tag: latest
 sto_image_tag: latest
 sg_core_image_tag: latest
 sg_bridge_image_tag: latest
 prometheus_webhook_snmp_image_tag: latest
 loki_operator_image_tag: latest
-new_operator_sdk_version: v1.5.0
+new_operator_sdk_version: v1.11.0
 new_go_version: 1.16.3
 namespace: service-telemetry
 
 # Set a default commit hash to clone for loki-operator to freeze
 # the operator developement.
-loki_operator_branch: 700e275
+loki_operator_branch: b8e9973
```

Comment:
spooky magic numbers

Reply:
the loki-operator is under development and there isn't a stable version released yet, so this is a commit hash